From 81f80551cdb44dd22a993055a295de7b2dafc8c1 Mon Sep 17 00:00:00 2001
From: Matthew Feickert <matthew.feickert@cern.ch>
Date: Sun, 14 Dec 2025 23:42:12 -0700
Subject: [PATCH] fix: Require Python 3.7+

* The features used in pyproject.toml were not accessible until
  Python 3.7 given the versions of Python that the necessary tools
  supported.
---
 pyproject.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pyproject.toml b/pyproject.toml
index 148aaa9..8fe24a1 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -11,7 +11,7 @@ authors = [
 ]
 description = "Parsing, manipulating, and visualising SUSY Les Houches Accord data"
 readme = "README.md"
-requires-python = ">=2.7"
+requires-python = ">=3.7"
 dependencies = ["tex2pix >=0.1.5"]
 keywords = ["supersymmetry", "susy", "slha", "simulation", "mass", "decay", "hep", "physics", "particle"]
 classifiers = [
-- 
2.52.0

